api/controllers/WebhooksController.js

```diff
@@ -21,13 +22,19 @@ async function _notifyNewDisaster(disaster) {
       /* eslint no-await-in-loop: "off" */
       const list = await List.findOne({ remote_id: operation.id });
       if (!list) {
+        logger.warn(
+          `[WebhooksController->_notifyNewDisaster] List with remote_id ${operation.id} not found`,
+        );
```

Comment:
I think this one should be `logger.error`

Reply:
Should be fixed by my last commit.